src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -337,6 +338,12 @@
 						"\tevent-projectile is arrow",
 						"\tdelete event-projectile")
 				.since("1.0");
+		
+		if(Skript.classExists("com.destroystokyo.paper.event.entity.ProjectileCollideEvent")) Skript.registerEvent("Projectile Collide", SimpleEvent.class, ProjectileCollideEvent.class, "projectile collide")
+			.description("Called when a projectile collides with an entity")
+			.examples("on projectile collide:",
+				"\tteleport shooter of event-projectile to event-entity")
+			.since("INSERT VERSION");
```

Comment:
Also missing a ``.requiredPlugins("Paper")``

Reply:
Added it